SU2_PY/OptimalPropeller.py

```diff
@@ -0,0 +1,371 @@
+# ==============================================================================================
```

Comment:
Put the SU2 license header above this one (it is a GPL requirement).

Reply:
Thank you for your comment, I have added the license header as on the other .py files.